### DISCUSSION TITLE
Selective restore on running system

### DISCUSSION THREAD

**Discussion by kingtat** [seedvault-app/seedvault#218] [Q&A] [answered]
I have not used android system backup before due to the limited storage options. I've used Helium though. Now, with seedvault as a system backup provider in lineageos, I decided to give the built-in backup/restore a try on a lineageos 17.1 installatioin.

The only place I encountered any mention of a restore operation was whilst installing the OS (lineageos).
Can I selectively restore specific system, or 3rd party, app data to a running android system?

**Comment by grote** [comment, marked as answer]
No that hasn't been implemented. It is an all or nothing for now.

**Reply by kingtat** [reply]
OK. Still, really appreciate the option of a local rather than cloud based restore system. Many thanks.
